features/login.feature

```diff
@@ -0,0 +1,22 @@
+Feature: Smoke
+
+  Scenario Outline: Change Currency on site
```

Comment:
Good idea to use Scenario Outline, but as this is not the final version I assume that you will add more scenarios for this part as if you have only 1 scenario then you don't need Scenario Outline